llvm/lib/CodeGen/RegAllocFast.cpp

```diff
@@ -1382,7 +1389,12 @@ void RegAllocFastImpl::allocateInstruction(MachineInstr &MI) {
   // - The "free def operands" step has to come last instead of first for tied
   //   operands and early-clobbers.
 
-  UsedInInstr.clear();
+  InstrGen += 2;
+  // In the event we ever get more than 2**31 instructions...
+  if (InstrGen == 0) {
```

Comment:
Is it worth putting an unlikely here?

Reply:
Didn't change assembly, but added it anyway for clarity.